Python/initconfig.c

```diff
@@ -100,9 +103,6 @@ static const char usage_3[] = "\
             when the interpreter displays tracebacks.\n\
          -X frozen_modules=[on|off]: whether or not frozen modules should be used.\n\
             The default is \"on\" (or \"off\" if you are running a local build).\n\
-\n\
---check-hash-based-pycs always|default|never:\n\
-    control how Python invalidates hash-based .pyc files\n\
```

Comment:
I'm guessing this was kept separate because it is a long arg.  I don't have much opinion on whether it should be kept separate or not. 🙂

Reply:
I don’t mind it staying at the end, but then I would move the item in the man page to match!

Reply:
Whatever works. :)

Reply:
I like the long options at the end of `--help` output, especially after the other PR that adds `--help-env`, `--help-xoptions` and `--help-all`).  I am also not reordering the man page: long options are on the last line of the synopsis, and in alphabetical order in the full doc, both of which are good.